src/ASM/ASMs2D.h

```diff
@@ -267,9 +267,10 @@ class ASMs2D : public ASMstruct, public ASM2D
   //! \param neighbor The neighbor patch
   //! \param[in] nedge Local edge index of neighbor patch, in range [1,4]
   //! \param[in] revers Indicates whether the two edges have opposite directions
+  //! \param[in] basis (ignored)
   //! \param[in] coordCheck False to disable coordinate checks (periodic connections)
   virtual bool connectPatch(int edge, ASMs2D& neighbor, int nedge,
-                            bool revers = false, bool coordCheck = true);
+                            bool revers = false, int basis = 0, bool coordCheck = true);
```

Comment:
kan evt. bare skrive int = 0, så slipper man dummy-parameteren i doxyen. Tilsv. for ASMs3D.